docs/content/s3.md

```diff
@@ -641,7 +642,7 @@ A simple solution is to set the `--s3-upload-cutoff 0` and force all the files t
 {{< rem autogenerated options start" - DO NOT EDIT - instead edit fs.RegInfo in backend/s3/s3.go then run make backenddocs" >}}
```

Comment:
Did you notice this? Its easy to miss, so I just wanted to make sure you did not manually edit text below here - which are not also reflected in the s3.go source and would therefore be lost.